kalliope/package.json

```diff
@@ -1,68 +1,110 @@
 {
-    "name": "kalliope",
-    "version": "0.0.1",
-    "author": "Kalliope Team",
-    "homepage": "https://kalliope-project.github.io/",
-    "private": true,
-    "scripts": {
-        "clean": "ionic-app-scripts clean",
-        "build": "ionic-app-scripts build",
-        "lint": "ionic-app-scripts lint",
-        "ionic:build": "ionic-app-scripts build",
-        "ionic:serve": "ionic-app-scripts serve"
+  "name": "kalliope",
+  "version": "0.0.1",
```

Comment:
0.5?

Reply:
Should we align the version number with Kalliope Core ?

Reply:
It's just that the PR is named v5...